11ty/definitions/pt_PT/cisgenero.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Cisgênero
```

Comment:
Like the "Transgênero" definition, "gênero" should be "género" everytime it is referred in the definition.

So here it should be "Cisgénero".